manager.spec.yaml

```diff
@@ -2624,9 +2624,18 @@ components:
       type: object
       properties:
         system_count:
```

Comment:
I think this change is going to break PDF generator. https://github.com/RedHatInsights/pdf-generator/blob/main/src/templates/vulnerability/executive-report/Template.tsx

I'd not change the `system_count` key but add it as a new key (and deprecate `system_count` later if needed when the PDF generator is updated in all envs).